### PR TITLE
some table additions

### PR DESCRIPTION
**Pull request by kweav**
Added some tables and info about general suggestions, if people are interested in being a guest speaker, and if they have any guest speaker recommendations.

### CODE REVIEW

**Comment by kweav** [comment]
I think this is the last of the content additions from survey questions. 

Looking for feedback on where I've added things and how they look/if it's the info you'd like. 

Next PR I'll fill out the home page, but figured it'd be best to have the content there first.

File: pages/general_info.Rmd [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+title: "General Comments and Suggestions"
+output: html_document
+---
+
+```{r echo=FALSE, message=FALSE, warning=FALSE}
+library(tidyverse)
+library(kableExtra)
+```
+
+## General Suggestions
```
Comment (carriewright11) [inline pages/general_info.Rmd]: Maybe don't need the title and the heading? Otherwise this LGTM!

**Review by carriewright11 — APPROVED** [review]

File: pages/general_info.Rmd [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`r data.frame(df$GenSuggestions[df$GenSuggestions %>% str_detect("meeting time|conflict")]) %>% drop_na() %>% nrow()` out of `r nrow(df %>% select(GenSuggestions) %>% drop_na())` the general suggestions/comments mentioned a time conflict.
```
Comment (kweav) [inline pages/general_info.Rmd]:
```suggestion
`r data.frame(df$GenSuggestions[df$GenSuggestions %>% str_detect("meeting time|conflict")]) %>% drop_na() %>% nrow()` out of `r nrow(df %>% select(GenSuggestions) %>% drop_na())` of the general suggestions/comments mentioned a time conflict.
```

File: pages/general_info.Rmd (hunk 1 above)
Comment (kweav) [inline pages/general_info.Rmd]:
```suggestion
```

**Pull request merged**